Testing/functional/tests/cypress/integration/ui/pages/vaccineCard.js

```diff
@@ -0,0 +1,545 @@
+
+        // Test and set Current Month
+        select(dobMonthSelector, monthNames[monthNumber]);
+        select(dovMonthSelector, monthNames[monthNumber]);
+
+        if (nextDay > 1) {
+            // Current Year and Month have been set. If next day is 1, it means previous.
+            // day was last day of current month. Next Day is associated with the current month.
+            // Test Future Day in current month does not exist.
+            selectNotExist(dobDaySelector, nextDay);
+            selectNotExist(dovDaySelector, nextDay);
+        }
+        //Test Current Day exists
+        selectExist(dobDaySelector, day);
+        selectExist(dovDaySelector, day);
+    });
+
+    it("Vaccination Card - DOB Year and DOV not entered via Click Enter - unauthenticated user", () => {
+        const dobMonth = "June";
+        const dobDay = "15";
+
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(Cypress.env("phn"));
+
+        select(dobMonthSelector, dobMonth);
+
+        select(dobDaySelector, dobDay);
+
+        clickVaccineCardEnterButton();
+
+        cy.get(feedbackDobIsRequiredSelector).should("be.visible");
+        cy.get(feedbackDovIsRequiredSelector).should("be.visible");
+    });
+
+    it("Vaccination Card - DOB Month and DOV not entered via Click Enter - unauthenticated user", () => {
+        const dobYear = "2021";
+        const dobDay = "15";
+
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(Cypress.env("phn"));
+
+        select(dobYearSelector, dobYear);
+
+        select(dobDaySelector, dobDay);
+
+        clickVaccineCardEnterButton();
+
+        cy.get(feedbackDobIsRequiredSelector).should("be.visible");
+        cy.get(feedbackDovIsRequiredSelector).should("be.visible");
+    });
+
+    it("Vaccination Card - DOB Day and DOV not entered via Click Enter - unauthenticated user", () => {
+        const dobYear = "2021";
+        const dobMonth = "June";
+
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(Cypress.env("phn"));
+
+        select(dobYearSelector, dobYear);
+
+        select(dobMonthSelector, dobMonth);
+
+        clickVaccineCardEnterButton();
+
+        cy.get(feedbackDobIsRequiredSelector).should("be.visible");
+        cy.get(feedbackDovIsRequiredSelector).should("be.visible");
+    });
+
+    it("Vaccination Card - DOV Year and DOB not entered via Click Enter - unauthenticated user", () => {
+        const dovMonth = "June";
+        const dovDay = "15";
+
+        cy.enableModules(vaccinationStatusModule);
+        cy.visit(vaccineCardUrl);
+
+        enterVaccineCardPHN(Cypress.env("phn"));
+
+        select(dovMonthSelector, dovMonth);
+
+        select(dovDaySelector, dovDay);
+
+        clickVaccineCardEnterButton();
+
+        cy.get(feedbackDobIsRequiredSelector).should("be.visible");
```

Comment:
Same note as above but reversed - I don't think we need to check date of birthdate in this test since it's focused on date of vaccination.  Applies to the following tests as well.

Reply:
I want to make sure all messages get displayed when DOB and DOV validation errors occur at the same time.